snap-nuke.sh

```diff
@@ -22,7 +21,7 @@ cat << EOF
 EOF
 
 while true; do
-    read -p "Do you wish to run the script? " yn
+    read -p "Do you wish to run the script? [y/n] " yn
```

Comment:
Why?

Reply:
So that the user knows what they're supposed to type. Without that, the prompt does not give any indication of which input it accepts. As it stands, the error message reads "Please answer yes or no.", but it only accepts `y`, `Y`, `n` or `N`.

It could use some clarification as well. Maybe something like "Please answer [y]es or [n]o."?

Reply:
Ahaha I misread the diff! I thought you were _removing_ `[y/n]`